### DISCUSSION TITLE
How I make this?

### DISCUSSION THREAD

**Discussion by icarosun** [autogluon/autogluon#1601] [Q&A] [answered]
" you can set the environment variable "AUTOGLUON_TEXT_TRAIN_WITHOUT_GPU=1" to force the model to use CPU for training."

The program not detected the gpu in ambient from google colab. I modify the ambient to GPU, but I can't run the quick start TextPrediction in colab. 

`print(get_cpu_count()) # 2
print(get_gpu_count()) # 1
print(get_gpu_count_mxnet()) # 0
`

**Comment by Innixma** [comment, marked as answer]
Please try the new v0.4 release and see if you run into the same problem. If so, please provide a link to the colab notebook.
